src/GraphQL.ApiTests/net50/GraphQL.approved.txt

```diff
@@ -2209,18 +2202,12 @@ namespace GraphQL.Types
     {
         string? Description { get; set; }
     }
-    public interface IProvideMetadata
-    {
-        System.Collections.Generic.Dictionary<string, object?> Metadata { get; }
-        TType GetMetadata<TType>(string key, System.Func<TType> defaultValueFactory);
-        TType GetMetadata<TType>(string key, TType defaultValue = default);
-        bool HasMetadata(string key);
-    }
+    public interface IProvideMetadata : GraphQL.Types.IMetadataBuilder { }
```

Comment:
What is the purpose of two identical interfaces?

Reply:
How else to provide writing methods to builders and not reading methods?

Reply:
I can unnecessarily move some methods from one interface to the other but why?

Reply:
> How else to provide writing methods to builders and not reading methods?

I do not understand.

Reply:
Understood.